engine/src/main/java/org/terasology/logic/players/LocalPlayerSystem.java

```diff
@@ -489,10 +491,13 @@ private void updateCamera(CharacterMovementComponent charMovementComp, Vector3f
             }
         }
 
-        if (charMovementComp.mode == MovementMode.GHOSTING) {
-            playerCamera.extendFov(24);
-        } else {
-            playerCamera.resetFov();
+        if(charMovementComp.mode != lastMode){
+            if (charMovementComp.mode == MovementMode.GHOSTING) {
```

Comment:
Ok, if we are brainstorming about this:

immaterial mode, intangible mode, wisp mode, incorporeal mode... oh... I like incorporeal.. let me find some synonyms...  bodyless, disembodied, discarnate (yuck!), spectral, spiritlike, ethereal, transcendent, supernatural, nonphysical...

I guess incorporeal is the one I like the best, because I feel most people instinctively would understand that something incorporeal can go through walls. That been said, incorporeal might twist the tongue of some teenager and spiritlike is a more dumbed down contender. Except that some spiritlike modes, i.e. in World of Warcraft, still abide to -some- of the laws of physics, i.e. they must follow the landscape rather than fly through it. 

So. Still vote for incorporeal mode. It seems very clear, unambiguos to me. But sticking to ghost (plus comment) or switching to spectator (or maybe observer? watcher? naah) would be fine too.